AIM/AIM/UI/Connectors/RootConnector.m

```diff
@@ -0,0 +1,50 @@
+//
+//  RootConnector.m
+//  AIM
+//
+//  Created by Oskar Szydlowski on 06.03.2018.
+//  Copyright © 2018 Oskar Szydlowski. All rights reserved.
+//
+
+#import "RootConnector.h"
+#import "TrackListConnector.h"
+#import "XMLOnAirInfoGateway.h"
+
+@interface RootConnector () {
+    
+    UIWindow *window;
+    TrackListConnector *connector;
+    UIViewController *viewController;
```

Comment:
This is an old C way of storing pointer in struct, so ARC doesn't work here. Is it intended to do it that way?